modules/base/include/modules/base/processors/dataexport.h

```diff
@@ -74,10 +74,10 @@ class DataExport : public Processor {
 template <typename DataType, typename PortType>
 DataExport<DataType, PortType>::DataExport()
     : Processor()
-    , port_("data")
-    , file_("file", "File name", "", "mesh")
-    , export_("export", "Export")
-    , overwrite_("overwrite", "Overwrite", false) {
+    , port_("data", "The data to export"_help)
+    , file_("file", "File name", "The file to save data to"_help, "", "mesh")
```

Comment:
that would be the "contentType" I did not change that, copied in the doc string. 
the content type should depend on with kid of data we want to export. It is probably left over since before this was a template. Now is should be something like ContentType<DataType>::value if we had such a construct...